lib/rucio/core/replica.py

```diff
@@ -855,7 +860,12 @@ def _build_list_replicas_pfn(
                             pass  # ATLAS HACK
                         else:
                             # don't forget to mangle gfal-style davs URL into generic https URL
-                            pfn = 'root://' + root_proxy_internal + '//' + pfn.replace('davs://', 'https://')
+                            pfn = f'root://{root_proxy_internal}//{pfn.replace("davs://", "https://")}'
+
+    simulate_multirange = get_rse_attribute(rse_id, 'simulate_multirange')
```

Comment:
there needs to be a fallback if simulate_multirange is not a number (and if it's a number it must be larger than 0)

Reply:
done. includes changing the corresponding tests